submenu.js

```diff
@@ -1,21 +1,17 @@
 const submenu = document.getElementById("nav__hamburger-submenu-wrapper");
-const hamburgerMenu = document.getElementById("nav__hamburger-menu");
-const hamburgerIcon = document.getElementById("nav__hamburger-icon");
 
-export function forwardbutton() {
+export function toggleSubmenu() {
     submenu.classList.toggle("--hidden");
-    hamburgerMenu.classList.toggle("--hidden");
 }
 
-export function submenuHide() {
+export function toggleSubmenuControl() {
     if (!submenu.classList.contains("--hidden")) {
-        forwardbutton()
+        toggleSubmenu();
     }
 }
 
-export function submenuControl() {
+export function toggleSubmenuControl2() {
     if (!submenu.classList.contains("--hidden")) {
         submenu.classList.toggle("--hidden");
-        hamburgerIcon.classList.toggle("Close-icon");
     }
```

Comment:
Search shows me only function declaration. Please remove it.
![image](https://user-images.githubusercontent.com/71021726/164718296-41988e24-863f-4548-8cff-f733e72052f2.png)